lib/cc/yaml/nodes/engine_list.rb

```diff
@@ -2,7 +2,30 @@ module CC
   module Yaml
     module Nodes
       class EngineList < OpenMapping
+        GENERIC_ERROR_MESSAGE = "engines should be a map of engine name to engine config"
```

Comment:
Fine by me - was suggesting it be capitalized because it is the first word in the sentence below